### ISSUE TITLE
New update: Hal Daumé III - new citations

### ISSUE THREAD

**Issue by sudocurse** [sudocurse/reading-list#13308]
From: scholaralerts-noreply@google.com<br>
Subject: Hal Daum&eacute; III - new citations<br>
First Attachment: https://ift.tt/19fNRe4<br>
Date: May 19, 2018 at 04:33AM<br>
<br>
<div style="font-family: arial,sans-serif; font-size: 13px; line-height: 16px; color: #222; width: 100%;">
<h3 style="font-weight: normal; margin: 0; font-size: 17px; line-height: 20px;">
<span style="font-size: 11px; font-weight: bold; color: #1a0dab; vertical-align: 2px;">[PDF]</span> <a href="http://scholar.google.com/scholar_url?url=https://arxiv.org/pdf/1805.05029&amp;hl=en&amp;sa=X&amp;scisig=AAGBfm1mePSWkbJN7ey-2wK6qXGFFNG8jA&amp;nossl=1&amp;oi=scholaralrt" class="gse_alrt_title" style="font-size: 17px; color: #1a0dab;">Multi-view Common Component Discriminant Analysis for Cross-view Classification</a>
</h3>
<div style="color: #006621;">X You, J Xu, W Yuan, XY Jing, D Tao, T Zhang&nbsp;- arXiv preprint arXiv:1805.05029, 2018</div>
<div class="gse_alrt_sni">Cross-view classification that means to classify samples from heterogeneous views<br>
is a significant yet challenging problem in computer vision. A promising approach to<br>
handle this problem is the multi-view subspace learning (MvSL), which intends to&nbsp;&hellip;</div>
<div style="width: auto;">
<table cellpadding="0" cellspacing="0" border="0">
<tbody>
<tr>
<td style="padding-right: 4px;"><a href="http://scholar.google.com/scholar_share?hl=en&amp;oi=scholaralrt&amp;ss=gp&amp;url=https://arxiv.org/pdf/1805.05029&amp;rt=Multi-view+Common+Component+Discriminant+Analysis+for+Cross-view+Classification&amp;scisig=AAGBfm0F_gf5nrAQu-Pu98er2uJoZxgiPA" style="text-decoration: none; display: inline-block; padding: 4px 8px 4px 0;"><img alt="Google+" src="http://scholar.google.com/intl/en/scholar/images/1x/gp-32.png" border="0" height="16" width="16" style="vertical-align: top;"></a></td>
<td style="padding-right: 4px;"><a href="http://scholar.google.com/scholar_share?hl=en&amp;oi=scholaralrt&amp;ss=fb&amp;url=https://arxiv.org/pdf/1805.05029&amp;rt=Multi-view+Common+Component+Discriminant+Analysis+for+Cross-view+Classification&amp;scisig=AAGBfm0F_gf5nrAQu-Pu98er2uJoZxgiPA" style="text-decoration: none; display: inline-block; padding: 4px 8px;"><img alt="Facebook" src="http://scholar.google.com/intl/en/scholar/images/1x/fb-32.png" border="0" height="16" width="16" style="vertical-align: top;"></a></td>
<td><a href="http://scholar.google.com/scholar_share?hl=en&amp;oi=scholaralrt&amp;ss=tw&amp;url=https://arxiv.org/pdf/1805.05029&amp;rt=Multi-view+Common+Component+Discriminant+Analysis+for+Cross-view+Classification&amp;scisig=AAGBfm0F_gf5nrAQu-Pu98er2uJoZxgiPA" style="text-decoration: none; display: inline-block; padding: 4px 8px;"><img alt="Twitter" src="http://scholar.google.com/intl/en/scholar/images/1x/tw-32.png" border="0" height="16" width="16" style="vertical-align: top;"></a></td>
</tr>
</tbody>
</table>
</div>
<br>
<h3 style="font-weight: normal; margin: 0; font-size: 17px; line-height: 20px;">
<span style="font-size: 11px; font-weight: bold; color: #1a0dab; vertical-align: 2px;">[PDF]</span> <a href="http://scholar.google.com/scholar_url?url=http://homepages.inf.ed.ac.uk/csutton/publications/deep_dnd_naacl2018.pdf&amp;hl=en&amp;sa=X&amp;scisig=AAGBfm2Ytdog7iBN1M5RdUUzU-1XDs3xgw&amp;nossl=1&amp;oi=scholaralrt" class="gse_alrt_title" style="font-size: 17px; color: #1a0dab;">Deep Dungeons and Dragons: Learning Character-Action Interactions from Role-Playing Game Transcripts</a>
</h3>
<div style="color: #006621;">A Louis, C Sutton</div>
<div class="gse_alrt_sni">An essential aspect to understanding narratives is to grasp the interaction between<br>
characters in a story and the actions they take. We examine whether computational<br>
models can capture this interaction, when both character attributes and actions are&nbsp;&hellip;</div>
<div style="width: auto;">
<table cellpadding="0" cellspacing="0" border="0">
<tbody>
<tr>
<td style="padding-right: 4px;"><a href="http://scholar.google.com/scholar_share?hl=en&amp;oi=scholaralrt&amp;ss=gp&amp;url=http://homepages.inf.ed.ac.uk/csutton/publications/deep_dnd_naacl2018.pdf&amp;rt=Deep+Dungeons+and+Dragons:+Learning+Character-Action+Interactions+from+Role-Playing+Game+Transcripts&amp;scisig=AAGBfm2XNCKDnKfcM_93d2ZI6hoHL0OKvA" style="text-decoration: none; display: inline-block; padding: 4px 8px 4px 0;"><img alt="Google+" src="http://scholar.google.com/intl/en/scholar/images/1x/gp-32.png" border="0" height="16" width="16" style="vertical-align: top;"></a></td>
<td style="padding-right: 4px;"><a href="http://scholar.google.com/scholar_share?hl=en&amp;oi=scholaralrt&amp;ss=fb&amp;url=http://homepages.inf.ed.ac.uk/csutton/publications/deep_dnd_naacl2018.pdf&amp;rt=Deep+Dungeons+and+Dragons:+Learning+Character-Action+Interactions+from+Role-Playing+Game+Transcripts&amp;scisig=AAGBfm2XNCKDnKfcM_93d2ZI6hoHL0OKvA" style="text-decoration: none; display: inline-block; padding: 4px 8px;"><img alt="Facebook" src="http://scholar.google.com/intl/en/scholar/images/1x/fb-32.png" border="0" height="16" width="16" style="vertical-align: top;"></a></td>
<td><a href="http://scholar.google.com/scholar_share?hl=en&amp;oi=scholaralrt&amp;ss=tw&amp;url=http://homepages.inf.ed.ac.uk/csutton/publications/deep_dnd_naacl2018.pdf&amp;rt=Deep+Dungeons+and+Dragons:+Learning+Character-Action+Interactions+from+Role-Playing+Game+Transcripts&amp;scisig=AAGBfm2XNCKDnKfcM_93d2ZI6hoHL0OKvA" style="text-decoration: none; display: inline-block; padding: 4px 8px;"><img alt="Twitter" src="http://scholar.google.com/intl/en/scholar/images/1x/tw-32.png" border="0" height="16" width="16" style="vertical-align: top;"></a></td>
</tr>
</tbody>
</table>
</div>
<br>
<h3 style="font-weight: normal; margin: 0; font-size: 17px; line-height: 20px;"><a href="http://scholar.google.com/scholar_url?url=https://link.springer.com/article/10.1007/s11128-018-1924-8&amp;hl=en&amp;sa=X&amp;scisig=AAGBfm1dPYXDGBuM-N7t42YrksqW4IPC_A&amp;nossl=1&amp;oi=scholaralrt" class="gse_alrt_title" style="font-size: 17px; color: #1a0dab;">Quantum-enhanced feature selection with forward selection and backward elimination</a></h3>
<div style="color: #006621;">Z He, L Li, Z Huang, H Situ&nbsp;- Quantum Information Processing, 2018</div>
<div class="gse_alrt_sni">Feature selection is a well-known preprocessing technique in machine learning,<br>
which can remove irrelevant features to improve the generalization capability of a<br>
classifier and reduce training and inference time. However, feature selection is time&nbsp;&hellip;</div>
<div style="width: auto;">
<table cellpadding="0" cellspacing="0" border="0">
<tbody>
<tr>
<td style="padding-right: 4px;"><a href="http://scholar.google.com/scholar_share?hl=en&amp;oi=scholaralrt&amp;ss=gp&amp;url=https://link.springer.com/article/10.1007/s11128-018-1924-8&amp;rt=Quantum-enhanced+feature+selection+with+forward+selection+and+backward+elimination&amp;scisig=AAGBfm2iyKV_Cr-bfS5vcL6i5Ifzn668sg" style="text-decoration: none; display: inline-block; padding: 4px 8px 4px 0;"><img alt="Google+" src="http://scholar.google.com/intl/en/scholar/images/1x/gp-32.png" border="0" height="16" width="16" style="vertical-align: top;"></a></td>
<td style="padding-right: 4px;"><a href="http://scholar.google.com/scholar_share?hl=en&amp;oi=scholaralrt&amp;ss=fb&amp;url=https://link.springer.com/article/10.1007/s11128-018-1924-8&amp;rt=Quantum-enhanced+feature+selection+with+forward+selection+and+backward+elimination&amp;scisig=AAGBfm2iyKV_Cr-bfS5vcL6i5Ifzn668sg" style="text-decoration: none; display: inline-block; padding: 4px 8px;"><img alt="Facebook" src="http://scholar.google.com/intl/en/scholar/images/1x/fb-32.png" border="0" height="16" width="16" style="vertical-align: top;"></a></td>
<td><a href="http://scholar.google.com/scholar_share?hl=en&amp;oi=scholaralrt&amp;ss=tw&amp;url=https://link.springer.com/article/10.1007/s11128-018-1924-8&amp;rt=Quantum-enhanced+feature+selection+with+forward+selection+and+backward+elimination&amp;scisig=AAGBfm2iyKV_Cr-bfS5vcL6i5Ifzn668sg" style="text-decoration: none; display: inline-block; padding: 4px 8px;"><img alt="Twitter" src="http://scholar.google.com/intl/en/scholar/images/1x/tw-32.png" border="0" height="16" width="16" style="vertical-align: top;"></a></td>
</tr>
</tbody>
</table>
</div>
<br>
<h3 style="font-weight: normal; margin: 0; font-size: 17px; line-height: 20px;">
<span style="font-size: 11px; font-weight: bold; color: #1a0dab; vertical-align: 2px;">[PDF]</span> <a href="http://scholar.google.com/scholar_url?url=https://arxiv.org/pdf/1805.05062&amp;hl=en&amp;sa=X&amp;scisig=AAGBfm0RbeDdO36Oh4WLwrJ_46PTWR4eRw&amp;nossl=1&amp;oi=scholaralrt" class="gse_alrt_title" style="font-size: 17px; color: #1a0dab;">Token-level and sequence-level loss smoothing for RNN language models</a>
</h3>
<div style="color: #006621;">M Elbayad, L Besacier, J Verbeek&nbsp;- arXiv preprint arXiv:1805.05062, 2018</div>
<div class="gse_alrt_sni">Despite the effectiveness of recurrent neural network language models, their<br>
maximum likelihood estimation suffers from two limitations. It treats all sentences that<br>
do not match the ground truth as equally poor, ignoring the structure of the output&nbsp;&hellip;</div>
<div style="width: auto;">
<table cellpadding="0" cellspacing="0" border="0">
<tbody>
<tr>
<td style="padding-right: 4px;"><a href="http://scholar.google.com/scholar_share?hl=en&amp;oi=scholaralrt&amp;ss=gp&amp;url=https://arxiv.org/pdf/1805.05062&amp;rt=Token-level+and+sequence-level+loss+smoothing+for+RNN+language+models&amp;scisig=AAGBfm137PKWDf5AHtnuF2rDVcJVVNMgLg" style="text-decoration: none; display: inline-block; padding: 4px 8px 4px 0;"><img alt="Google+" src="http://scholar.google.com/intl/en/scholar/images/1x/gp-32.png" border="0" height="16" width="16" style="vertical-align: top;"></a></td>
<td style="padding-right: 4px;"><a href="http://scholar.google.com/scholar_share?hl=en&amp;oi=scholaralrt&amp;ss=fb&amp;url=https://arxiv.org/pdf/1805.05062&amp;rt=Token-level+and+sequence-level+loss+smoothing+for+RNN+language+models&amp;scisig=AAGBfm137PKWDf5AHtnuF2rDVcJVVNMgLg" style="text-decoration: none; display: inline-block; padding: 4px 8px;"><img alt="Facebook" src="http://scholar.google.com/intl/en/scholar/images/1x/fb-32.png" border="0" height="16" width="16" style="vertical-align: top;"></a></td>
<td><a href="http://scholar.google.com/scholar_share?hl=en&amp;oi=scholaralrt&amp;ss=tw&amp;url=https://arxiv.org/pdf/1805.05062&amp;rt=Token-level+and+sequence-level+loss+smoothing+for+RNN+language+models&amp;scisig=AAGBfm137PKWDf5AHtnuF2rDVcJVVNMgLg" style="text-decoration: none; display: inline-block; padding: 4px 8px;"><img alt="Twitter" src="http://scholar.google.com/intl/en/scholar/images/1x/tw-32.png" border="0" height="16" width="16" style="vertical-align: top;"></a></td>
</tr>
</tbody>
</table>
</div>
<br>
<h3 style="font-weight: normal; margin: 0; font-size: 17px; line-height: 20px;">
<span style="font-size: 11px; font-weight: bold; color: #1a0dab; vertical-align: 2px;">[PDF]</span> <a href="http://scholar.google.com/scholar_url?url=https://arxiv.org/pdf/1805.04893&amp;hl=en&amp;sa=X&amp;scisig=AAGBfm0UoVk7YZks76DbxaIMsxTImSQv9g&amp;nossl=1&amp;oi=scholaralrt" class="gse_alrt_title" style="font-size: 17px; color: #1a0dab;">Neural Coreference Resolution with Deep Biaffine Attention by Joint Mention Detection and Mention Clustering</a>
</h3>
<div style="color: #006621;">R Zhang, CN Santos, M Yasunaga, B Xiang, D Radev&nbsp;- arXiv preprint arXiv&nbsp;&hellip;, 2018</div>
<div class="gse_alrt_sni">Coreference resolution aims to identify in a text all mentions that refer to the same<br>
real-world entity. The state-of-the-art end-to-end neural coreference model considers<br>
all text spans in a document as potential mentions and learns to link an antecedent&nbsp;&hellip;</div>
<div style="width: auto;">
<table cellpadding="0" cellspacing="0" border="0">
<tbody>
<tr>
<td style="padding-right: 4px;"><a href="http://scholar.google.com/scholar_share?hl=en&amp;oi=scholaralrt&amp;ss=gp&amp;url=https://arxiv.org/pdf/1805.04893&amp;rt=Neural+Coreference+Resolution+with+Deep+Biaffine+Attention+by+Joint+Mention+Detection+and+Mention+Clustering&amp;scisig=AAGBfm1tAeYr46m91dI6YDOzOIMAC_wZ-A" style="text-decoration: none; display: inline-block; padding: 4px 8px 4px 0;"><img alt="Google+" src="http://scholar.google.com/intl/en/scholar/images/1x/gp-32.png" border="0" height="16" width="16" style="vertical-align: top;"></a></td>
<td style="padding-right: 4px;"><a href="http://scholar.google.com/scholar_share?hl=en&amp;oi=scholaralrt&amp;ss=fb&amp;url=https://arxiv.org/pdf/1805.04893&amp;rt=Neural+Coreference+Resolution+with+Deep+Biaffine+Attention+by+Joint+Mention+Detection+and+Mention+Clustering&amp;scisig=AAGBfm1tAeYr46m91dI6YDOzOIMAC_wZ-A" style="text-decoration: none; display: inline-block; padding: 4px 8px;"><img alt="Facebook" src="http://scholar.google.com/intl/en/scholar/images/1x/fb-32.png" border="0" height="16" width="16" style="vertical-align: top;"></a></td>
<td><a href="http://scholar.google.com/scholar_share?hl=en&amp;oi=scholaralrt&amp;ss=tw&amp;url=https://arxiv.org/pdf/1805.04893&amp;rt=Neural+Coreference+Resolution+with+Deep+Biaffine+Attention+by+Joint+Mention+Detection+and+Mention+Clustering&amp;scisig=AAGBfm1tAeYr46m91dI6YDOzOIMAC_wZ-A" style="text-decoration: none; display: inline-block; padding: 4px 8px;"><img alt="Twitter" src="http://scholar.google.com/intl/en/scholar/images/1x/tw-32.png" border="0" height="16" width="16" style="vertical-align: top;"></a></td>
</tr>
</tbody>
</table>
</div>
<br>
<div style="line-height: 16px; border-top: 1px solid #bdbdbd;">&nbsp;</div>
<div>
<div><a href="http://scholar.google.com/scholar_alerts?view_op=list_alerts&amp;email_for_op=postmillenial%40gmail.com&amp;alert_id=j92EtR6u_wYJ&amp;hl=en" style="display: inline-block; text-decoration: none; font-family: arial,sans-serif; font-size: 11px; font-size: 13px; line-height: 21px; padding: 3px 0; color: #1a0dab; border-top: 1px solid transparent; border-bottom: 1px solid transparent; border-radius: 3px;"><span>List my alerts</span></a></div>
<div><a href="http://scholar.google.com/scholar_alerts?view_op=cancel_alert_options&amp;email_for_op=postmillenial%40gmail.com&amp;alert_id=j92EtR6u_wYJ&amp;hl=en" style="display: inline-block; text-decoration: none; font-family: arial,sans-serif; font-size: 11px; font-size: 13px; line-height: 21px; padding: 3px 0; color: #1a0dab; border-top: 1px solid transparent; border-bottom: 1px solid transparent; border-radius: 3px;"><span>Cancel alert</span></a></div>
</div>
<p style="margin: 8px 0 24px 0; color: #666;">This alert is sent by Google Scholar. Google Scholar is a service by Google.</p>
</div>
<br>
<br>
via Gmail
